### DISCUSSION TITLE
fit model using REML

### DISCUSSION THREAD

**Discussion by jbigman-NOAA** [vast-lib/tinyVAST#57] [Q&A]
I would like to fit a few models with REML vs ML. I pasted a simple model below - can you help me edit it so it fits with REML instead of ML? 

Thanks!

```

sem = "
  n <-> n, spatial_sd
"

mod <- 
    tinyVAST(
    data = d,
    formula = log_wt ~ 0 + age_f,
    family = gaussian(),
    sem = sem, 
    space_column = c("X", "Y"), 
    spatial_graph = mesh_p,
    time_column = "year",
    control = tinyVASTcontrol(getsd=FALSE))
```

**Comment by James-Thorson** [comment]
Looks like I haven't built in a way to pass a different character-vector of random effects, or a higher-level Boolean switch for REML.  Before I add one (presumably the latter) ... do you mind switching to the dev branch if I add the feature there?

**Reply by jbigman-NOAA** [reply]
I don't mind switching to the dev branch. Thanks Jim!

**Reply by James-Thorson** [reply]
OK, I just added a `tinyVASTcontrol( reml=TRUE )` flag and checked that it works with `print.tinyVAST` but haven't done a ton of exploration ... do you mind trying it out, and comparing `fit$opt$par` with and without REML to confirm that fixed effects are affected.  

I'm closing for now but happy to follow up on this thread.

**Comment by jbigman-NOAA** [comment]
Thanks Jim! Will test it out.
